app/models/teacher.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: teachers
+#
+#  id         :uuid             not null, primary key
+#  first_name :string           not null
+#  mentor     :boolean          default(FALSE)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Teacher < ApplicationRecord
+end
```

Comment:
I can see you've gone with the name "Teacher" rather than "Mentor", which means we need to specify `source: "teacher"` on the `Claim#mentors` association.

I'm wondering: what led you to choose this name?

In the [data model document](https://github.com/DFE-Digital/itt-mentor-services/blob/beaa69d2273f9db16b52e341ed04d519675b3517/docs/data-model.md), we have a concept of a "Mentor" and a "Trainee" being two independent entities. Both have a TRN so they can be looked up in DQT – and I'm wondering if this is the reason you've named the model "Teacher"?

However conceptually Mentors and Trainees two different things – they have different meanings within the service. And in fact, based on our current definition of MVP, we won't actually need Trainee yet.

So I'm wondering if we'd be better off just calling this "Mentor"?

Reply:
I named it teacher because from what I understand is that DQT doesn't know which people are mentors and which are teachers, so I assumed we will import all teachers from DQT and have to distinguish between teachers and mentors so a boolean field on it was doing the job. I can change it to mentors though, don't mind

Reply:
I guess it's a bit misleading as I'm not actually using the `mentor` boolean, I'll make some changes so that it's actually used

Reply:
Sure that makes sense. I didn't know all of this. I'll change it to mentor